target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -29,6 +29,9 @@ ipq40xx_setup_interfaces()
 	zyxel,nbg6617)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
 		;;
+	zyxel,wsq50)
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" "wan"
```

Comment:
There are already boards using the same config, just add yours there, there is no need to duplicate code.

Reply:
Will do 👍

Reply:
This has been fixed - extra code removed and used another config